cfpb/cfgov-refresh
# Update external link/icons logic

Anything that's not an internal page, including other .gov pages, should show the external link icon.  Anything that's leaving .gov pages should take the user to the external-site page.

## Testing

- Go to [http://localhost:8000/cfpb-ombudsman/ombudsman-faqs/](http://localhost:8000/cfpb-ombudsman/ombudsman-faqs/) & confirm that under "Do the other federal financial agencies have an ombudsman?", the links show the external icon but DO NOT take you to the external-site page.
- Click on the facebook icon & confirm that it takes you to the external-site page. (I can't find an example of a link that's not already an icon, but if you can, confirm that it shows the external icon as well). 
- View any cf.gov links and confirm they neither show the icon or take you to the external-site page.

## Review

- @kave @rosskarchner @kurtw 

## Screenshots

<img width="919" alt="screen shot 2016-03-29 at 12 54 19 pm" src="https://cloud.githubusercontent.com/assets/354591/14121710/652136d6-f5ad-11e5-9ca0-4c5b6102ffb8.png">

## Checklist
